src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -679,6 +680,7 @@ private JabRefPreferences() {
         defaults.put(FX_THEME, Theme.BASE_CSS);
 
         setLanguageDependentDefaultValues();
+        USER_NAME = getUserName();
```

Comment:
Initializing it here has the disadvantage that it slows down the start of JabRef. Thus, I would propose to put it in the `getUser` method below:
```java
    public String getUser() {
         if (StringUtil.isNotBlank(userName))
              return userName;
         
          try {
            userName = get(DEFAULT_OWNER) + '-' + InetAddress.getLocalHost().getHostName();
            return userName;
        } catch (UnknownHostException ex) {
            LOGGER.debug("Hostname not found.", ex);
            return get(DEFAULT_OWNER);
        }
```

Reply:
changed to lazy initialization, please pay attention to log level.